clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1646,16 +1646,10 @@ public void resume(Collection<TopicPartition> partitions) {
     }
 
     /**
-     * Get the last offset for the given partitions. The last offset of a partition is the offset of the next
-     * message produced into this partition, i.e. the offset of the last available message + 1. If messages have
-     * never been written to the the partition, the offset returned will be 0.
-     *
-     * <p>
-     * This method does not change the current consumer position of the partitions.
```

Comment:
Looks like we lost this message. Do you think it's still worth mentioning?

Reply:
Thanks, @hachikuji. I put this message back.